components/Button.js

```diff
@@ -10,54 +10,104 @@ export function Button({ type, variant, onClick, children }) {
 
 const DefaultButton = styled.button`
   border: none;
+  border-radius: 3px;
   box-shadow: rgba(60, 64, 67, 0.3) 0px 1px 2px 0px,
     rgba(60, 64, 67, 0.15) 0px 1px 3px 1px;
-  border-radius: 3px;
-  background-color: oldlace;
+  background-color: var(--fourth-color);
+  font-size: 16px;
+  font-family: "Poppins-Light";
 
   &:hover {
-    background-color: lightgrey;
+    cursor: pointer;
   }
 
   ${({ variant }) =>
     variant === "cancel" &&
     css`
       margin: 10px;
-      font-weight: 600;
-      align-self: flex-end;
+      font-size: 16px;
+      color: var(--second-font-color);
+
+      :hover {
+        background-color: #e6e6e6;
```

Comment:
As you are using this color multiple times, I suggest to use a css variable as you did for your font color.